### ISSUE TITLE
Syncplay and MPC-HC hanging when switching video via sharedplaylist

### ISSUE THREAD

**Issue by RFOyashiro** [Syncplay/syncplay#664]
**Describe the bug**
When using a shared playlist, when a video finishes and syncplay switches to the next one, both MPC-HC and Syncplay hang and don't answer anymore until both are killed through task manager.

**To Reproduce**
1. Load multiple videos in a shared playlist
2. Start and finish a video (that is not the last of the playlist)
3. Wait for the video to switch to the next one
4. Both programs hang

**Expected behavior**
MPC-HC open the file correctly and Syncplay run as normal, wait everyone ready

**Version and platform:**
 - OS: Windows 10 64-bit
 - Syncplay version and build type: Syncplay 1.7.1 release 109
 - Media player and version:  MPC-HC (64-bit) 2.1.4 (c82061aef). Also happens on MPC-HC (64-bit) 2.0.0

**Additional context**
- This bug does not happen when manually switching video OR when finishing the last video of the shared playlist.
- File .syncplay.log is filled with the same 2 lines repeating over and over (File too big, reaching 682 Mb after 1 hour usage) :
```
WNDPROC return value cannot be converted to LRESULT
TypeError: WPARAM is simple, so must be an int object (got NoneType)
```
- Here is the [Syncplay.log](https://github.com/Syncplay/syncplay/files/13997368/Syncplay.log) file after having to kill the program through task manager.
- This issue has been somehow appearing since about 2 or 3 months without updating any of both program (Syncplay 1.7.1 and MPC-HC 2.0.0). Trying to update to the newest MPC-HC version did not resolve the problem.
- If needed, here is my processor and graphic card : AMD Ryzen 5 2600 Six-Core Processor 3.40 GHz, Radeon RX 570 Series

**Comment by Et0h** [comment]
Thanks for reporting this issue. I can confirm that Syncplay does not work properly with MPC-HC. As MPC-HC has no official maintainers and alternative media players exist that people can use instead , I have decided it is time to officially discontinue MPC-HC and MPC-BE support and recommend that Windows users move to mpv.net instead. See https://github.com/Syncplay/syncplay/issues/666 for details.

**Comment by Et0h** [comment]
I think I've fixed the issue. You can download the latest development build with the patch at https://github.com/Syncplay/syncplay/actions/runs/7732159464 and let me know if it works

P.S. As there is no official Syncplay maintainer for MPC-HC support my recommendation is that people move to mpv/mpv.net (which in any case has better Syncplay integration/support).

**Comment by Et0h** [comment]
This fix was included in Syncplay 1.7.2 - I'll assume it worked. Let me know if it didn't.
